framework/src/modules/chain/processor/processor.js

```diff
@@ -65,7 +65,10 @@ class Processor {
 		await this._processGenesis(genesisBlock, blockProcessor, {
 			skipSave: false,
 		});
-		await blockProcessor.init.run();
+		await this.blocksModule.init();
+		for (const processor of Object.values(this.processors)) {
+			await processor.init.run();
+		}
```

Comment:
Are these changes necessary under the scope of this PR?

Reply:
that's a good point. It was not necessary, but at the same time, bft was initialized from root, but blocks were initialized in each. (which should be opposite) so I just fixed it...

Reply:
Moved to #4297